airdrops/index_v2.json

```diff
@@ -252,6 +252,24 @@
                 "chain_id": 1,
                 "decimals": 18
             }
+        },
+        "zk": {
+            "csv_path": "airdrops/zk.csv",
+            "csv_hash": "ee22fada7eef5cfb1e308797e0d0fb5b92deec35a9c5ff992b24958acd9bab62",
+            "asset_identifier": "eip155:324/erc20:0x5A7d6b2F92C77FAD6CCaBd7EE0624E64907Eaf3E",
+            "url": "https://claim.zknation.io/",
+            "name": "ZK",
+            "icon": "zksync.jpg",
+            "cutoff_time": 1735862400,
+            "has_decoder": false,
+            "new_asset_data": {
+                "asset_type": "EVM_TOKEN",
+                "address": "0x5A7d6b2F92C77FAD6CCaBd7EE0624E64907Eaf3E",
+                "name": "zkSync",
```

Comment:
Where did you get the name from? I was double checking and seems to be `ZKsync`
![image](https://github.com/rotki/data/assets/5068010/3c0a11c0-9681-4e40-b47f-162166f3b695)